src/components/pages/labs/categories/categories.jsx

```diff
@@ -0,0 +1,54 @@
+import clsx from 'clsx';
+import { navigate } from 'gatsby';
+import PropTypes from 'prop-types';
+import React from 'react';
+
+const Categories = ({ categories, currentCategory, className }) => {
+  const handleCategoryClick = (slug) => (event) => {
+    event.preventDefault();
+    navigate(slug, {
+      state: { preventScroll: true },
+    });
+  };
+
+  return (
+    <section className={clsx('container-sm', className)}>
```

Comment:
In the section, there should be the class `safe-paddings`. And only in the next div element we leverage `container`. Please see other components for examples (e.g. [home page](https://deploy-preview-438--ebpfio.netlify.app/))
<img width="324" alt="image" src="https://github.com/ebpf-io/ebpf.io-website/assets/48465000/9e38bdeb-363a-402a-804c-64daf11336ca">